cunumeric/coverage.py

```diff
@@ -250,6 +250,37 @@ def clone_module(
                 cast(AnyCallable, value), mod_name, attr, reporting=reporting
             )
             new_globals[attr] = wrapped
+            if isinstance(value, lgufunc):
+                wrapped.reduce = implemented(  # type: ignore
```

Comment:
Wrapping these as implemented means we don't get the performance warning when falling back. I would expect something like this:
```
print(cn.add.accumulate(cn.arange(8)))
```
to print out the "cuNumeric has not implemented ... and is falling back" warning. We could wrap them as unimplemented here, or maybe on each individual method in ufunc.py.

Reply:
typo, should have been `unimplemented`

Reply:
actually, `reduce` is implemented, isn't it (it is the only one, AFAICT) `reduce` specifically does
```
        array = convert_to_cunumeric_ndarray(array)

        if self._red_code is None:
            raise NotImplementedError(
                f"reduction for {self} is not yet implemented"
            )
        if not isinstance(where, bool) or not where:
            raise NotImplementedError(
                "the 'where' keyword is not yet supported"
            )

        # NumPy seems to be using None as the default axis value for scalars
        if array.ndim == 0 and axis == 0:
            axis = None

        # TODO: Unary reductions still need to be refactored
        return array._perform_unary_reduction(
            self._red_code,
            array,
            axis=axis,
            dtype=dtype,
            out=out,
            keepdims=keepdims,
            initial=initial,
            where=where,
        )
```

Reply:
marked all as `unimplemented` except `reduce` in https://github.com/nv-legate/cunumeric/pull/834/commits/de80e2a0786dc3141cf2ef525d893ab92a9b7997